p2p/peer_behaviour.go

```diff
@@ -0,0 +1,89 @@
+package p2p
+
+type ErrorBehaviourPeer int
+
+const (
+	ErrorBehaviourUnknown = iota
+	ErrorBehaviourBadMessage
+	ErrorBehaviourMessageOutofOrder
+)
+
+type GoodBehaviourPeer int
+
+const (
+	GoodBehaviourVote = iota
+	GoodBehaviourBlockPart
+)
+
+// PeerBehaviour provides an interface for reactors to signal the behaviour
+// of peers synchronously to other components.
+type PeerBehaviour interface {
```

Comment:
Does it make sense to move the interface in a separate package?

Reply:
Only if we bundle all things peer together I'd say. But as long as peer is part of p2p it's not really helping much.